.aicoe-ci.yaml

```
@@ -6,3 +6,8 @@ build:
   registry-org: open-services-group
   registry-project: peribolos-as-service
```

```suggestion
  registry-project:  peribolos-as-a-service
```

I have changed the quay repo, to be the same as the GitHub repo
 quay.io/open-services-group/peribolos-as-a-service